s32/mcux/devices/S32K146/S32K146_features.h

```diff
@@ -301,4 +301,8 @@
 /* @brief Has Oscillator Enable(bitfield CR[OSCE]). */
 #define FSL_FEATURE_RTC_HAS_NO_CR_OSCE (1)
 
+/* ADC module features */
+
+#define FSL_FEATURE_ADC12_CONVERSION_CONTROL_COUNT (24)
```

Comment:
add all `FSL_FEATURE_ADC12_*` features and give explicit configuration values for this device, don't leave implicit configuration. Also please add the documentation string (you can copy paste an existing device definition and update the values)

Reply:
ok